packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -193,6 +193,8 @@ function ChangePassword({
     <View
       as="form"
       className={ComponentClassName.ChangePassword}
+      // React.FormEventHandler<HTMLFormElement> does not accept async function
+      // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Rather than disable linting rules should we expand the `onSubmit` prop to allow async callbacks?

Reply:
Hmmm, hard to say if it is not in scope since this PR is touching the file and disabling linting rules?